test/evergreen.yml

```diff
@@ -4099,10 +4099,10 @@ buildvariants:
     - name: ".stress-test-ppc-1"
     - name: ".stress-test-ppc-2"
 
-- name: ubuntu1804-zseries
-  display_name: "~ Ubuntu 18.04 zSeries"
+- name: rhel80-zseries
+  display_name: "~ RHEL80 zSeries"
   run_on:
-  - ubuntu1804-zseries-test
+  - rhel80-zseries-test
```

Comment:
I notice `rhel80-zseries-test` is an alias name to `rhel83-zseries-small`. Though I'm confident both would work, I'd hope to seek some advice on which one is recommended to use in evergreen.yml. Can we check it with the Build team?

Reply:
I've left a question for the build team on this but I noticed the other tasks use the aliased names.